src/apps/Hosting/HostingDialog.js

```diff
@@ -94,8 +97,16 @@ const CreateHostingDialog = React.createClass({
             <Dialog.SidebarSection>
               Hosting allows you to manage, deploy and publish websites using Syncano platform.
             </Dialog.SidebarSection>
+            <Dialog.SidebarSection title="Hosting label">
+              You can set the name of your hosting.
+            </Dialog.SidebarSection>
+            <Dialog.SidebarSection title="Domains">
+              {`Define the domains. Each name must be unique within an Instance. You can also
```

Comment:
i think we should also mention about the structure of the domain, `https://${currentInstanceName}--{domain}.syncano.site`